ci/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM docker:dind
+
+# set labels
+LABEL maintainer="Nightah"
+
+# set application versions
+ARG ARCH="amd64"
+ARG BUILDKITE_VERSION="3.17.0"
+ARG OVERLAY_VERSION="v1.22.1.0"
+
+# environment variables
+ENV PS1="$(whoami)@$(hostname):$(pwd)$ " \
+HOME="/buildkite" \
+TERM="xterm"
+
+# set runtime variables
+ENV BUILDKITE_AGENT_CONFIG=/buildkite/buildkite-agent.cfg \
+GOPATH="/buildkite/.go"
+
+# add local files
+COPY root/ /
+
+# modifications
+RUN \
+ echo "**** Install Authelia CI pre-requisites ****" && \
+   echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
+   echo "@edget http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
+   apk add --no-cache \
+     bash \
+     ca-certificates \
+     coreutils \
+     chromium \
+     chromium-chromedriver \
+     curl \
+     docker-compose \
+     git \
+     hub@edget \
```

Comment:
why is it edget instead of edge?

Reply:
Because of this:
https://github.com/clems4ever/authelia/blob/4bd491c3ce9b4cf6e87837b24c2be48aa033daa0/ci/Dockerfile#L26-L27
some of the packages are in different edge repos so `@edge` referers to edge/community and `@edget` refers to edge/testing.

[Hub](https://pkgs.alpinelinux.org/packages?name=hub&branch=edge) specifically resides in edge/testing.

Reply:
perfect, thanks.